thirdparty/runtime_environments.go

```diff
@@ -96,3 +96,49 @@ func (c *RuntimeEnvironmentsClient) getOSInfo(ctx context.Context, amiID string,
 	}
 	return osInfo, nil
 }
+
+// ImageInfo represents information about an image with its AMIID and creation date.
+type ImageInfo struct {
+	AMIID        string
+	CreationDate string
+}
+
+// DistoHistoryFilter represents the filtering arguments for getHistory. The Distro field is required and the other fields are optional.
+type DistroHistoryFilter struct {
```

Comment:
Maybe we can end with `FilterOptions` to be consistent with the other structs in this file:
```suggestion
type DistroHistoryFilterOptions struct {
```